network-api/networkapi/management/commands/fix_redirects.py

```diff
@@ -0,0 +1,56 @@
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+from wagtail.contrib.redirects.models import Redirect
+from wagtail.models import Locale
+
+# Define the list of valid values to check against
+LOCALES = Locale.objects.all().values_list("language_code", flat=True)
+
+
+def check_path(url_path):
+
+    # Extract the first part of the URL path
+    path_parts = url_path.split("/")
+    first_part = path_parts[1] if len(path_parts) > 1 else ""
+
+    # Check if the first part of the URL path is in the valid list of values
+    if first_part in LOCALES:
+        # do nothing
+        return True
+    else:
+        # Create a redirect for this path
+        return False
+
+
+class Command(BaseCommand):
+    help = "Creates redirects for each locale for each redirect if it doesn't exist."
+
+    def handle(self, *args, **options):
+        redirects = Redirect.objects.all()
+
+        for redirect in redirects:
+            has_locale_redirect = check_path(url_path=redirect.old_path)
+            if not has_locale_redirect:
+                # create a new redirect for each locale, it will be a copy
+                # of the original redirect but with the locale prefix
+                for locale in LOCALES:
+                    # first check if the redirect already exists, this will happen
+                    # if we run this code more than once
+                    if not redirects.filter(old_path=f"/{locale}{redirect.old_path}"):
```

Comment:
What do you think about using a `.exists()` in the end here? I think its supposed to prevent the database from having to return the real objects but can only return the boolean response. 

https://docs.djangoproject.com/en/4.2/ref/models/querysets/#exists

Reply:
added in a5a3ce03e713b132657c47ee34bfa2233c13d109